link/src/main/java/com/stripe/android/link/ui/LinkButton.kt

```diff
@@ -156,21 +159,26 @@ private fun RowScope.SignedInButtonContent(email: String) {
 @Suppress("UnusedReceiverParameter")
 @Composable
 private fun RowScope.SignedOutButtonContent() {
+    val text = stringResource(id = R.string.stripe_pay_with_link)
+
     val iconizedText = buildAnnotatedString {
-        append("Pay with") // TODO(jaynewstrom) Link: Add localization
-        append(" ")
+        append(text.substringBefore(LINK_BRAND_NAME))
```

Comment:
In other strings, we use tags to replace the image. See https://github.com/stripe/stripe-android/blob/9792026bdd45e0d30ac657d5144eb0304ee1f284/payments-ui-core/res/values/strings.xml#L3-L6

That _might_ be less error prone?

Reply:
I prefer this! I'll see if we can create another string translation for something like this following this PR.